Character.cpp

```diff
@@ -22,39 +25,72 @@
         return this->ATK;
     }
 
+    //JSON parse method for creating a Character object based on a given JSON input file
+    Character* Character::parseUnit(const std::string path) 
+    {
+        std::ifstream f(path);
+        //We check if the file given as input exists
+        if (f.good()) 
+        {
+            //We read the whole file into a string variable using ifstream and stringstream
+            //We do this because since we'll use the split method anyway, a counter variable holding which row we're currently reading is not needed
+            //This saves us a few addition and divide operations here
+            std::stringstream s;
+            s << f.rdbuf();
+            std::string fileContents = s.str();
```

Comment:
Így is jó, de ez stringstream nélkül is megoldható: `std::getline(f, fileContents, '\0');`